spec/application_spec.rb

```diff
@@ -152,6 +152,64 @@ module Core
           newapp.plan_name.must_equal 'some_other_plan'
         end
 
+        it 'save active application' do
+          @app.activate
+          @app.save
+          newapp = Application.load @app.service_id, @app.id
+          newapp.active?.must_equal true
+        end
+
+        it 'save inactive application' do
+          @app.deactivate
+          @app.save
+          newapp = Application.load @app.service_id, @app.id
+          newapp.active?.must_equal false
+        end
```

Comment:
Should we add a test checking that the app is active when we don't activate or deactivate it?

Reply:
added test to save application with no explicit state (when we don't activate or deactivate). On loading, it should be active